jha/src/views/enrolment/SuppBenInfoPage.vue

```diff
@@ -424,6 +422,8 @@ export default {
       if (this.selectedAttendantNursingRecipients.includes("spouse")){
         this.selectedAttendantNursingRecipients.splice(this.selectedAttendantNursingRecipients.indexOf("spouse"), 1);
       }
+    } else {
+      this.selectOptionsFamilyMembers.filter(option => {return option.id === 'spouse';})[0].disabled = false;
```

Comment:
Not a big fan of this line, but you can simplify this a tiny bit by doing:
```javascript
this.selectOptionsFamilyMembers.filter(option => option.id === 'spouse')[0].disabled = false;
```
This is more personal preference, but it's up to you.